notebook-lm-clone/src/memory/memory_layer.py

```
@@ -0,0 +1,326 @@
+        )
+        self.external_memory = ExternalMemory(storage=self.user_storage)
+        
+        logger.info(f"NotebookMemoryLayer initialized for user {user_id}, session {session_id}")
+    
+    def _setup_user_and_session(self, create_new_session: bool):
+        try:
+            # Ensure user exists
+            try:
+                self.zep_client.user.get(self.user_id)
+                logger.info(f"Using existing user: {self.user_id}")
+            except:
```

Ruff (0.13.1)

Do not use bare `except`

(E722)